modules/conversion-center/classes/render/contact-buttons-render-base.php

```diff
@@ -246,11 +245,11 @@ protected function render_contact_links(): void {
 						'number' => $icon['contact_icon_number'] ?? '',
 						'username' => $icon['contact_icon_username'] ?? '',
 						'email_data' => [
-							'contact_icon_mail' => $icon['contact_icon_mail'],
+							'contact_icon_mail' => $icon['contact_icon_mail'] ?? '',
 							'contact_icon_mail_subject' => $icon['contact_icon_mail_subject'] ?? '',
 							'contact_icon_mail_body' => $icon['contact_icon_mail_body'] ?? '',
 						],
-						'viber_action' => $icon['contact_icon_viber_action'],
+						'viber_action' => $icon['contact_icon_viber_action'] ?? '',
 					];
 
 					$formatted_link = $this->get_formatted_link( $link, 'contact_icon' );
```

Comment:
@mserino do we also need to check for existence of $link here before passing it to the formatting function?

Reply:
@willhowat (I am going to reply to this comment only, it applies to all the others 😅 ) there is no need to check the existence of `$link` because I'm "manually" generating it a few lines above, and I am checking the existence of each element of the array. So, worst case scenario it's an array with empty strings, but it's never null.